<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="B_Upd_Table">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Quest">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить условие</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B_And">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>И</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B_Else">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Или</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Text_ed">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="B_Clear">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сбросить Условия</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>461</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
